practice-two/src/pages/Analytics/Analytics.jsx

```diff
@@ -126,6 +142,7 @@ const Analytics = () => {
       {selectedCustomer && isShowProfileInfo && (
         <ProfileInfo selectedCustomer={selectedCustomer} />
       )}
+      {isShowForm && <FormValidation handleShowForm={handleShowForm} />}
```

Comment:
update name of props is onShowForm. Don't use handle for props

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/86b3aa64-8a36-4935-a59e-6a1b4b757c3a)